Listenbrainz/UI/Components/TopBar.swift

```diff
@@ -12,13 +12,15 @@ struct TopBar: View {
 
   var body: some View {
     HStack {
-      Image("listenBrainzDark")
-        .resizable()
-        .frame(width: 40, height: 40)
-        .padding(12)
-        .cornerRadius(20)
-        .clipShape(Circle())
+      HStack(spacing:0){
+        LBIcon2()
+          .fill(Color.color_2)
```

Comment:
You can use color names like primary, secondary instead of color 1 or 2

Reply:
done